calico_containers/adapter/datastore.py

```diff
@@ -1,15 +1,18 @@
 from collections import namedtuple
+import copy
 import json
+import os
+import re
+
 import etcd
 from etcd import EtcdKeyNotFound, EtcdException
 from netaddr import IPNetwork, IPAddress, AddrFormatError
-import os
-import copy
 
 ETCD_AUTHORITY_DEFAULT = "127.0.0.1:4001"
 ETCD_AUTHORITY_ENV = "ETCD_AUTHORITY"
 
 # etcd paths for Calico
+# TODO: modify string constants to accept orchestrator_id, replacing "docker" hardcoding
```

Comment:
Can you expand this comment a bit. Just be clear that we're not cleaning this up just yet because not all of the methods in datastore.py are aware of the orchestrator ID yet.